### PR TITLE
Did some mappings in RedstoneWireBlock

### PR DESCRIPTION
**Pull request by LostQuasar**

### CODE REVIEW

**Review by YanisBft — APPROVED** [review]

**Comment by LostQuasar** [comment]
I feel like a better name could be used here but getReceivedPower makes the most sense, perhaps a change should be made to the method im world to be more clear about its purpose which is to find the highest power on all horizontal directions

File: mappings/net/minecraft/block/RedstoneWireBlock.mapping [hunk 1]
```diff
@@ -28,6 +28,9 @@ CLASS net/minecraft/class_2457 net/minecraft/block/RedstoneWireBlock
 		ARG 1 world
 		ARG 2 state
 		ARG 3 pos
+	METHOD method_27842 geReceivedRedstonePower (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)I
```
Comment (haykam821) [inline mappings/net/minecraft/block/RedstoneWireBlock.mapping]:
```suggestion
	METHOD method_27842 getReceivedRedstonePower (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)I
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Review by liach — APPROVED** [review]

**Review by haykam821 — APPROVED** [review]
The rest of the methods can be fully mapped later.

**Comment by liach** [comment]
Hmm, I doubt the rest of these methods may just be lambdas. Anyways look great, ready to go

**Pull request merged**